packs/classfeatures/weapon-expertise.json

```
@@ -32,13 +32,79 @@
                 "predicate": [
                     {
                         "or": [
+                            "class:bard",
                             "class:champion",
+                            "class:inventor",
                             "class:investigator",
-                            "class:magus"
+                            "class:magus",
+                            "class:swashbuckler"
                         ]
                     }
                 ],
                 "value": 2
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.proficiencies.attacks.weapon-base-alchemical-bomb.rank",
+                "predicate": [
+                    "class:alchemist"
+                ],
+                "value": 2
+            },
+            {
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    "feature:weapon-innovation",
+                    "class:inventor",
+                    "item:id:{actor|flags.pf2e.innovationId}"
+                ]
+            },
+            {
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    {
+                        "gte": [
+                            "item:proficiency:rank",
+                            2
+                        ]
+                    },
+                    "class:swashbuckler"
+                ]
+            },
+            {
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    "hunted-prey",
+                    {
+                        "or": [
+                            "item:category:simple",
+                            "item:category:martial",
+                            "item:category:unarmed"
+                        ]
+                    }
+                ]
+            },
+            {
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.{actor|class.slug}.Title"
```

```suggestion
                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.Title"
```
I actually don't think we need this at all. the ones that have different names shouldn't get condensed.